src/main/java/frc/robot/RobotContainer.java

```diff
@@ -225,7 +225,7 @@ private void configureBindings() {
         drivetrain.setDefaultCommand(new Drive(drivetrain, joyleft::getY, joyleft::getX, joyright::getX, drivetrain::getDriveMode));
         Command rampMoveAndSpin = CommandUtil.race(
                 "AdvancedAutoShoot",
-                new RampFollow(ramp, ()-> drivetrain.getAlignable(), ()-> drivetrain.getPose()),
+                new BoringRampFollow(ramp, drivetrain),
```

Comment:
Maybe I don't understand how this works....The alignable is being set by pressing an holding one button and then the aligning itself is done by holding a different button? the setting and following should, I think, be done together.